common/build.gradle

```diff
@@ -0,0 +1,174 @@
+
+    testImplementation group: 'org.apache.hive', name: 'hive-jdbc', version: '1.2.2', withoutServers
+
+    compileOnly group: 'org.apache.hadoop', name: 'hadoop-hdfs', version: '3.3.2', withoutServers
+    compileOnly group: 'org.apache.hadoop', name: 'hadoop-common', version: '3.3.2', withoutServers
+
+    implementation group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
+    //compileOnly group: 'org.apache.commons', name: 'commons-text', version: '1.7'
+//    jmh group: 'org.neo4j', name: 'neo4j-lucene-index', version: neo4jVersionEffective
+//    jmh group: 'org.neo4j', name: 'neo4j-kernel', version: neo4jVersionEffective, classifier: "tests"
+
+    testImplementation group: 'org.xmlunit', name: 'xmlunit-core', version: '2.2.1'
+    testImplementation group: 'com.github.adejanovski', name: 'cassandra-jdbc-wrapper', version: '3.1.0'
+
+    testImplementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-csv', version: '2.13.2'
+    testImplementation group: 'org.skyscreamer', name: 'jsonassert', version: '1.5.0'
+    testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.13.2'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.2.0'
+
+    compileOnly group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.6.1'
+    testImplementation group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.6.1', {
+        exclude group: 'com.google.guava', module: 'guava'
+    }
+
+    implementation group: 'com.google.guava', name: 'guava', version: '31.0.1-jre'
+
+    implementation group: 'org.apache.arrow', name: 'arrow-vector', version: '7.0.0', {
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+        exclude group: 'io.netty', module: 'netty-common'
+    }
+    implementation group: 'org.apache.arrow', name: 'arrow-memory-netty', version: '7.0.0', {
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'io.netty', module: 'netty-common'
+        exclude group: 'io.netty', module: 'netty-buffer'
+    }
+    testImplementation group: 'org.apache.arrow', name: 'arrow-vector', version: '7.0.0'
+    testImplementation group: 'org.apache.arrow', name: 'arrow-memory-netty', version: '7.0.0'
+
+    configurations.all {
+        exclude group: 'org.slf4j', module: 'slf4j-nop'
+        exclude group: 'ch.qos.logback', module: 'logback-classic'
+    }
+}
+
+
+// tweaks for CI
+if (System.env.CI == 'true' || System.env.TEAMCITY_VERSION != null) {
+  allprojects {
+    tasks.withType(GroovyCompile) {
+      groovyOptions.fork = false
+    }
+    tasks.withType(Test) {
+      // containers (currently) have 2 dedicated cores and 4GB of memory
+      maxParallelForks = 2
+      minHeapSize = '128m'
+    }
+  }
+}
```

Comment:
Do you think we could remove this completely in the 3 common/core/processor build.gradle files? 

I'm not sure the tests are particularly parallelizable unfortunately, even when run locally.

Reply:
I've removed it, so let's see, but I'm not a fan of introducing unnecessary changes in PRs and this was pre-existing